ci/base-images/cdxgen/Dockerfile.python313

```diff
@@ -0,0 +1,43 @@
+FROM ghcr.io/cyclonedx/bci-python313:master
+
+LABEL maintainer="CycloneDX" \
+      org.opencontainers.image.authors="Team AppThreat <cloud@appthreat.com>" \
+      org.opencontainers.image.source="https://github.com/CycloneDX/cdxgen" \
+      org.opencontainers.image.url="https://github.com/CycloneDX/cdxgen" \
+      org.opencontainers.image.version="rolling" \
+      org.opencontainers.image.vendor="CycloneDX" \
+      org.opencontainers.image.licenses="Apache-2.0" \
+      org.opencontainers.image.title="cdxgen" \
+      org.opencontainers.image.description="Rolling image with cdxgen SBOM generator for Python 3.13 apps" \
+      org.opencontainers.docker.cmd="docker run --rm -v /tmp:/tmp -p 9090:9090 -v $(pwd):/app:rw -t ghcr.io/cyclonedx/cdxgen-python313:v11 -r /app --server"
+
+ARG NODE_VERSION=20.19.2
```

Comment:
I think we can use Node 24 with these images. Node 20 is usually only required with Python 3.10 or 3.11.